docs/python3-mac-installation.md

```diff
@@ -0,0 +1,15 @@
+# Latest Python3 installation on macOS
```

Comment:
What's the problem we're solving? Do people not know how to install Python? If we agree we need something, we can just include a link, as in, 

Install the [version of Python](https://www.python.org/downloads/) for your OS.